### ISSUE TITLE
Incorrect ordering is passing for reasons I don't understand

### ISSUE THREAD

**Issue by alex** [PyCQA/flake8-import-order#9]
As far as I know, this should fail because the imported names aren't sorted alphabetically, but for me it doesn't show any warnings

``` py
from sqlalchemy import Integer, MetaData, Column

Integer, MetaData, Column,
```
